### PR TITLE
A few small changes

### PR DESCRIPTION
**Pull request by nwf**
Mostly to elftoolchain build.  I found it necessary to run "make obj" on my FreeBSD VM, for example.  The rest are mostly superficial.

### CODE REVIEW

**Comment by arichardson** [comment]
I believe I fixed the Cmake issue, is that change still required for you? Otherwise looks good to me

File: pycheribuild/projects/elftoolchain.py [hunk 1]
```diff
@@ -91,8 +91,9 @@ def compile(self, **kwargs):
             version = get_program_version(Path(self.make_args.command), ("-V", "MAKE_VERSION"), regex=b"(.+)")[0]
             # The version of bmake shipped with ubuntu doesn't handle depedencies correctly
             if version > 20170101:
-                statusUpdate("Note: Working around old version of bmake: ", version)
                 is_old_broken_bmake = False
+            else :
```
Comment (arichardson) [inline pycheribuild/projects/elftoolchain.py]: removeextra space before the `:`

**Comment by nwf** [comment]
Trade you: dropped the cmake hunk but added one to disk_image.  The issue there is that wget can't create its temporary index files over smbfs, which leads to sadness for those of us crazy enough to have extra-files so mounted.

**Comment by arichardson** [comment]
Thanks!

**Pull request merged**